<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>195</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Verdana</family>
    <pointsize>-1</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Kitsu Publisher</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*-----QWidget-----*/
QWidget
{
	background-color: #36393f;
	color: #ffffff;
	border-color: #051a39;
	font: 10px &quot;Verdana&quot;;

}

QWidget#main_frame{
border: 1px solid #fd8008;
}

/*-----QLabel-----*/
QLabel
{
	background-color: transparent;
	color: #ffffff;

}


QLabel::disabled
{
	background-color: transparent;
	color: #656565;

}


/*-----QMenuBar-----*/
QMenuBar
{
	background-color: #36393f;
	color: #ffffff;
	border-color: #051a39;

}


QMenuBar::disabled
{
	background-color: #404040;
	color: #656565;
	border-color: #051a39;

}


QMenuBar::item
{
    background-color: transparent;

}


QMenuBar::item:selected
{
    background-color: #fd8008;
    border: 1px solid #41cd52;

}


QMenuBar::item:pressed
{
    background-color: #b55d00;
    border: 1px solid #000;
    margin-bottom: -1px;
    padding-bottom: 1px;

}


/*-----QMenu-----*/
QMenu
{
    background-color: #36393f;
    border: 1px solid;
    color: #ffffff;

}


QMenu::separator
{
    height: 1px;
    background-color: #ff9500;
    color: #ffffff;
    padding-left: 4px;
    margin-left: 10px;
    margin-right: 5px;

}


QMenu::item
{
    min-width : 150px;
    padding: 3px 20px 3px 20px;

}


QMenu::item:selected
{
    background-color: #b55d00;
    color: #ffffff;

}


QMenu::item:disabled
{
    color: #262626;
}


/*-----QToolTip-----*/
QToolTip
{
	border : 1px solid #000000;
	background-color: #26264f;
	color: #ffffff;
	border-color: #051a39;

}



/*-----QPushButton-----*/
QPushButton
{
	background-color: #fd8008;
	color: #000000;
	font-weight: 600;
	border-style: solid;
	border-width: 1px;
	border-radius: 3px;
	border-color: #051a39;
	padding: 5px;

}


QPushButton::disabled
{
	background-color: #404040;
	color: #656565;
	border-color: #051a39;

}


QPushButton::hover
{
	background-color: #ff9d00;
	color: #ffffff;
	border-style: solid;
	border-width: 1px;
	border-radius: 3px;
	border-color: #051a39;
	padding: 5px;

}


QPushButton::pressed
{
	background-color: #b55d00;
	color: #ffffff;
	border-style: solid;
	border-width: 1px;
	border-radius: 3px;
	border-color: #051a39;
	padding: 5px;

}


/*-----QToolButton-----*/
QToolButton
{
	background-color: #fd8008;
	color: #ffffff;
	border-width: 1px;
	border-radius: 3px;
	border-color: #051a39;
	padding: 3px;

}


QToolButton::disabled
{
	background-color: #404040;
	color: #656565;
	border-color: #051a39;

}


QToolButton::hover
{
	background-color: #ff9d00;
	color: #ffffff;
	border-width: 1px;
	border-radius: 3px;
	border-color: #051a39;
	padding: 3px;

}


QToolButton::pressed
{
	background-color: #b55d00;
	color: #ffffff;
	border-width: 1px;
	border-radius: 3px;
	border-color: #051a39;
	padding: 3px;

}


/*-----QComboBox-----*/
QComboBox
{
   
    border: 1px solid;
	border-color:#262626;
    border-radius: 3px;
    padding-left: 6px;
    color: #ffffff;
    height: 20px;

}


QComboBox::disabled
{
	background-color: #404040;
	color: #656565;
	border-color: #051a39;

}


QComboBox:hover
{
    background-color: #ff9d00;

}


QComboBox:on
{
    background-color: #b55d00;

}


QComboBox QAbstractItemView
{
    background-color: #383838;
    color: #ffffff;
    border: 1px solid black;
    selection-background-color: #b55d00;
    selection-color: #ffffff;
    outline: 0;

}

QComboBox::down-arrow
{
    image: url(icons:arrow-down.svg);
    width: 8px;
    height: 8px;
}



QComboBox::drop-down
{
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;
    border-left-width: 0px;
    border-left-color: darkgray;
    border-left-style: solid; 
    border-top-right-radius: 3px; 
    border-bottom-right-radius: 3px;

}






/*-----QSpinBox &amp; QDoubleSpinBox &amp; QDateTimeEdit-----*/
QSpinBox, 
QDoubleSpinBox,
QDateTimeEdit
{
	background-color: #525251;
	color: #ffffff;
	border: 1px solid #051a39;
	border-radius: 3px;
	padding : 2px;

}


QSpinBox::disabled, 
QDoubleSpinBox::disabled,
QDateTimeEdit::disabled
{
	background-color: #404040;
	color: #656565;
	border-color: #051a39;

}


QSpinBox:hover, 
QDoubleSpinBox::hover,
QDateTimeEdit::hover
{
    background-color: #626262;
    border: 1px solid #fd8008;
    color:  #fff;
    padding: 2px

}


QSpinBox::up-button, QSpinBox::down-button,
QDoubleSpinBox::up-button, QDoubleSpinBox::down-button,
QDateTimeEdit::up-button, QDateTimeEdit::down-button
{
    background-color: #fd8008;
	border-radius: 2px;

}


QSpinBox::disabled, 
QDoubleSpinBox::disabled,
QDateTimeEdit::disabled
{
	background-color: #404040;
	color: #656565;
	border-color: #051a39;

}


QSpinBox::up-button:hover, QSpinBox::down-button:hover,
QDoubleSpinBox::up-button:hover, QDoubleSpinBox::down-button:hover,
QDateTimeEdit::up-button:hover, QDateTimeEdit::down-button:hover
{
    background-color: #ff9d00;
    border: 1px solid #ff9d00;

}


QSpinBox::up-button:disabled, QSpinBox::down-button:disabled,
QDoubleSpinBox::up-button:disabled, QDoubleSpinBox::down-button:disabled,
QDateTimeEdit::up-button:disabled, QDateTimeEdit::down-button:disabled
{
	background-color: #404040;
	color: #656565;
	border-color: #051a39;

}


QSpinBox::up-button:pressed, QSpinBox::down-button:pressed,
QDoubleSpinBox::up-button:pressed, QDoubleSpinBox::down-button::pressed,
QDateTimeEdit::up-button:pressed, QDateTimeEdit::down-button::pressed
{
    background-color: #b55d00;
    border: 1px solid #b55d00;

}



QSpinBox::down-arrow,
QDoubleSpinBox::down-arrow,
QDateTimeEdit::down-arrow
{
    image: url(icons:arrow-down.svg);
    width: 7px;

}


QSpinBox::up-arrow,
QDoubleSpinBox::up-arrow,
QDateTimeEdit::up-arrow
{
    image: url(icons:arrow-up.svg);
    width: 7px;

}



/*-----QLineEdit-----*/
QLineEdit
{
	background-color: #525251;
	color: #ffffff;
	border-width: 1px;
	border-radius: 3px;
	border-color: #051a39;
	padding: 2px;

}


QLineEdit::disabled
{
	background-color: #404040;
	color: #656565;
	border-width: 1px;
	border-radius: 3px;
	border-color: #051a39;
	padding: 2px;

}


/*-----QTextEdit-----*/
QTextEdit
{
	/* background-color: rgb(18, 18, 18); */
	color: #cecece;
	border-color: #051a39;

}


QTextEdit::disabled
{
	background-color: #404040;
	color: #656565;
	border-color: #051a39;

}


/*-----QGroupBox-----*/
QGroupBox 
{
    border: 1px solid;
    border-color: #fd8008;
    margin-top: 22px;
	font-weight: 600;

}


QGroupBox::title  
{
    background-color: #fd8008;
    color: #000000;
	font-size: 6pt;
	font-weight: 600;
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 5px;
	border-top-left-radius: 3px;
	border-top-right-radius: 3px;

}


QGroupBox::title::disabled
{
	background-color: #404040;
	color: #656565;
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 5px;
	border-top-left-radius: 3px;
	border-top-right-radius: 3px;

}


/*-----QRadioButton-----*/
QRadioButton::indicator::unchecked
{ 
	border: 2px inset gray; 
	border-radius: 5px; 
	background-color:  #fff;
	width: 9px; 
	height: 9px; 

}


QRadioButton::indicator::unchecked:hover
{ 
	border: 2px solid #fd8008; 
	border-radius: 5px; 
	background-color:  #fff;
	width: 9px; 
	height: 9px; 

}


QRadioButton::indicator::checked
{ 
	border: 2px inset darkgray; 
	border-radius: 5px; 
	background-color: #b55d00; 
	width: 9px; 
	height: 9px; 

}


QRadioButton::disabled
{
	color: #656565;

}


QRadioButton::indicator:disabled
{
	background-color: #656565;
	color: #656565;
    border: 2px solid #656565;

}


/*-----QTableView &amp; QTableWidget-----*/
QTableView
{
    background-color: #242526;
    border: 1px solid #32414B;
    color: #f0f0f0;
    gridline-color: #ff9900;
    outline : 0;

}


QTableView::disabled
{
    background-color: #242526;
    border: 1px solid #32414B;
    color: #656565;
    gridline-color: #656565;
    outline : 0;

}


QTableView::item:hover 
{
    background-color: #26264f;
    color: #f0f0f0;

}


QTableView::item:selected 
{
    background-color: #1a1b1c;
    border: 2px solid #b55d00;
    color: #F0F0F0;

}


QTableView::item:selected:disabled
{
    background-color: #1a1b1c;
    border: 2px solid #525251;
    color: #656565;

}


QTableCornerButton::section
{
    background-color: #505050;
    color: #fff;

}


QHeaderView::section
{
    background-color: #525251;
    color: #fff;
    text-align: left;
	padding: 4px;
	
}


QHeaderView::section:disabled
{
    background-color: #525251;
    color: #656565;

}


QHeaderView::section:checked
{
    color: #fff;
    background-color: #b55d00;

}


QHeaderView::section:checked:disabled
{
    color: #656565;
    background-color: #525251;

}


QHeaderView::section::vertical::first,
QHeaderView::section::vertical::only-one
{
    border-top: 1px solid #353635;

}


QHeaderView::section::vertical
{
    border-top: 1px solid #353635;

}


QHeaderView::section::horizontal::first,
QHeaderView::section::horizontal::only-one
{
    border-left: 1px solid #353635;

}


QHeaderView::section::horizontal
{
    border-left: 1px solid #353635;

}


/*-----QTabWidget-----*/
QTabBar::tab
{
	background-color: #262626;
	color: #ffffff;
	border-style: solid;
	border-width: 1px;
	border-top-left-radius: 3px;
	border-top-right-radius: 3px;
	border-color: #051a39;
	padding: 5px;

}


QTabBar::tab:disabled
{
	background-color: #656565;
	color: #656565;

}


QTabWidget::pane 
{
	background-color: #262626;
	color: #ffffff;
    border: 1px solid;
    border-color: #fd8008;

}


QTabBar::tab:selected
{
    background-color: #fd8008;
	color: #ffffff;
	border-style: solid;
	border-width: 1px;
	border-top-left-radius: 3px;
	border-top-right-radius: 3px;
	border-color: #051a39;
	padding: 5px;

}


QTabBar::tab:selected:disabled
{
	background-color: #404040;
	color: #656565;

}


QTabBar::tab:!selected 
{
    background-color: #262626;

}


QTabBar::tab:!selected:hover 
{
    background-color: #ff9d00;

}


QTabBar::tab:top:!selected 
{
    margin-top: 3px;

}


QTabBar::tab:bottom:!selected 
{
    margin-bottom: 3px;

}


QTabBar::tab:top, QTabBar::tab:bottom 
{
    min-width: 8ex;
    margin-right: -1px;
    padding: 5px 10px 5px 10px;

}


QTabBar::tab:top:selected 
{
    border-bottom-color: none;

}


QTabBar::tab:bottom:selected 
{
    border-top-color: none;

}


QTabBar::tab:top:last, QTabBar::tab:bottom:last,
QTabBar::tab:top:only-one, QTabBar::tab:bottom:only-one 
{
    margin-right: 0;

}


QTabBar::tab:left:!selected 
{
    margin-right: 3px;

}


QTabBar::tab:right:!selected
{
    margin-left: 3px;

}


QTabBar::tab:left, QTabBar::tab:right 
{
    min-height: 8ex;
    margin-bottom: -1px;
    padding: 10px 5px 10px 5px;

}


QTabBar::tab:left:selected 
{
    border-left-color: none;

}


QTabBar::tab:right:selected 
{
    border-right-color: none;

}


QTabBar::tab:left:last, QTabBar::tab:right:last,
QTabBar::tab:left:only-one, QTabBar::tab:right:only-one 
{
    margin-bottom: 0;

}


/*-----QSlider-----*/
QSlider::groove:horizontal 
{
	background-color: transparent;
	height: 5px;

}


QSlider::sub-page:horizontal 
{
	background-color: #fd8008;

}


QSlider::add-page:horizontal 
{
	background-color: #666765;

}


QSlider::handle:horizontal 
{
	background-color: #fd8008;
	width: 14px;
	margin-top: -6px;
	margin-bottom: -6px;
	border-radius: 3px;

}


QSlider::handle:horizontal:hover 
{
	background-color: #fd8008;
	border-radius: 3px;

}


QSlider::sub-page:horizontal:disabled 
{
	background-color: #bbb;
	border-color: #999;

}


QSlider::add-page:horizontal:disabled 
{
	background-color: #eee;
	border-color: #999;

}


QSlider::handle:horizontal:disabled 
{
	background-color: #eee;
	border: 1px solid #aaa;
	border-radius: 3px;

}


QSlider::groove:vertical 
{
	background-color: transparent;
	width: 5px;

}


QSlider::sub-page:vertical 
{
	background-color: #fd8008;

}


QSlider::add-page:vertical 
{
	background-color: #666765;

}


QSlider::handle:vertical 
{
	background-color: #fd8008;
	height: 14px;
	margin-left: -6px;
	margin-right: -6px;
	border-radius: 3px;

}


QSlider::handle:vertical:hover 
{
	background-color: #fd8008;
	border-radius: 3px;

}


QSlider::sub-page:vertical:disabled 
{
	background-color: #bbb;
	border-color: #999;

}


QSlider::add-page:vertical:disabled 
{
	background-color: #eee;
	border-color: #999;

}


QSlider::handle:vertical:disabled 
{
	background-color: #eee;
	border: 1px solid #aaa;
	border-radius: 3px;

}


/*-----QDial-----*/
QDial
{
	background-color: #fd8008;

}


QDial::disabled
{
	background-color: #404040;

}

/*-----QScrollBar-----*/
QScrollBar:horizontal
{
    border: 1px solid #222222;
    background-color: #3d3d3d;
    height: 13px;
    margin: 0px 16px 0 16px;

}


QScrollBar::handle:horizontal
{
    background: #fd8008;
    min-height: 20px;

}


QScrollBar::add-line:horizontal
{
    border: 1px solid #1b1b19;
    background-color: #fd8008;
    width: 14px;
    subcontrol-position: right;
    subcontrol-origin: margin;

}


QScrollBar::sub-line:horizontal
{
    border: 1px solid #1b1b19;
    background-color: #fd8008;
    width: 14px;
    subcontrol-position: left;
    subcontrol-origin: margin;

}


QScrollBar::right-arrow:horizontal
{
    image: url(icons:arrow-right.svg);
    width: 6px;
    height: 6px;

}


QScrollBar::left-arrow:horizontal
{
    image: url(icons:arrow-left.svg);
    width: 6px;
    height: 6px;

}


QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal
{
    background: none;

}


QScrollBar:vertical
{
    background-color: #3d3d3d;
    width: 13px;
    margin: 16px 0 16px 0;
    border: 1px solid #222222;

}


QScrollBar::handle:vertical
{
    background-color: #fd8008;
    min-height: 20px;

}


QScrollBar::add-line:vertical
{
    border: 1px solid #1b1b19;
    background-color: #fd8008;
    height: 14px;
    subcontrol-position: bottom;
    subcontrol-origin: margin;

}


QScrollBar::sub-line:vertical
{
    border: 1px solid #1b1b19;
    background-color: #fd8008;
    height: 14px;
    subcontrol-position: top;
    subcontrol-origin: margin;

}


QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical
{
    background: none;

}


QScrollBar::up-arrow:vertical
{
    image: url(icons:arrow-up.svg);
    width: 6px;
    height: 6px;

}


QScrollBar::down-arrow:vertical
{
    image: url(icons:arrow-down.svg);
    width: 6px;
    height: 6px;

}

/*-----QProgressBar-----*/
QProgressBar
{
    border: 1px solid #222222;
    background-color: #3d3d3d;
    text-align: center;
    color: #ffffff;
    font-size: 8px;
    /* font-weight: bold; */
    padding: 1px;
    border-radius: 3px;
}
QProgressBar::chunk
{
    background-color: #fd8008;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="frame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">border-color: rgb(253, 128, 8);
background-color: rgb(253, 128, 8);
color: rgb(0, 0, 0);
font-weight:600</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>8</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>KITSU CONNECTION SETTINGS</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="exit_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">

/*-----QPushButton-----*/
QPushButton
{
	background-color: rgba(255, 255, 255, 0);
	color: rgb(255, 255, 255);
	border-style:none;
}

QPushButton::hover
{
	color: #ffffff;
}

QPushButton::pressed
{
	background-color: rgba(255, 255, 255, 0);
	color: #ffffff;
}

</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../icons/x.svg</normaloff>../icons/x.svg</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="main_frame" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_10">
      <property name="spacing">
       <number>8</number>
      </property>
      <property name="leftMargin">
       <number>12</number>
      </property>
      <property name="topMargin">
       <number>12</number>
      </property>
      <property name="rightMargin">
       <number>12</number>
      </property>
      <property name="bottomMargin">
       <number>12</number>
      </property>
      <item>
       <layout class="QFormLayout" name="formLayout_3">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::ExpandingFieldsGrow</enum>
        </property>
        <property name="horizontalSpacing">
         <number>8</number>
        </property>
        <property name="verticalSpacing">
         <number>8</number>
        </property>
        <property name="leftMargin">
         <number>12</number>
        </property>
        <property name="topMargin">
         <number>12</number>
        </property>
        <property name="rightMargin">
         <number>12</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="styleSheet">
           <string notr="true">font-weight:600;</string>
          </property>
          <property name="text">
           <string>SERVER URL</string>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::NoTextInteraction</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="t_url">
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>https://kitsu.exemple.com</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_7">
          <property name="styleSheet">
           <string notr="true">font-weight:600;</string>
          </property>
          <property name="text">
           <string>USERNAME</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="t_user">
          <property name="placeholderText">
           <string>user@exemple.com</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_8">
          <property name="styleSheet">
           <string notr="true">font-weight:600;</string>
          </property>
          <property name="text">
           <string>PASSWORD</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="t_pwd">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="status_c">
          <property name="styleSheet">
           <string notr="true">color:#808795;</string>
          </property>
          <property name="text">
           <string>STATUS</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QPushButton" name="save_button">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string notr="true">SAVE AND CONNECT</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>exit_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
